stock_picking_batch_creation/__manifest__.py

```diff
@@ -14,6 +14,7 @@
     "application": False,
     "installable": True,
     "depends": [
+        "base_partition",
```

Comment:
@lmarion-source This addon is not available on OCA....

Reply:
Moreover it's not used at all...

Reply:
My bad, I forgot to remove it!